examples/CA/003_CA_ORU_R01_CDPH_produced_0_initial_message.hl7

```diff
@@ -1,4 +1,4 @@
-MSH|^~\&|SISGDSP|SISGDSP|SISHIERECEIVER^11903029^L,M,N|^^L,M,N|20240212103049||ORU^R01^ORU_R01|243408787|T|2.5.1
+MSH|^~\&|SISGDSP|SISGDSP|SISHIERECEIVER^11903029^L,M,N|^^L,M,N|20240212103049||ORU^R01^ORU_R01|243408787|D|2.5.1

```

Comment:
Does this change need to be made in the `final` messages too? Like `003_CA_ORU_R01_CDPH_produced_3_hl7_translation_final.hl7` still has `T` in msh-11, and I assume the other `final` messages are like that

Reply:
I think it can be okay to have the `final` ones be like...a snapshot in time, as long as it's noted that they are. But I think if we don't update that value, they're not actually the right output for those inputs? Since I don't think MSH-11 changes through the workflow. It might be nice to either note when they were last generated, or have a way to regenerate them automatically. The nice part of having the files more static is there's fewer surprises, but having them more up to date makes them more valid for tests

Reply:
I think regenerating them automatically is a good idea. It should be straightforward to create a bash script to do that using hurl. Though we'd need to manually get and overwrite the FHIR files from the local azure storage to update those

Reply:
I'll create a new PR to add a script that will automate those updates as much as possible